starfish/core/intensity_table/intensity_table_coordinates.py

```diff
@@ -1,46 +1,63 @@
+from typing import Mapping, Optional
+
 import numpy as np
 import xarray as xr
 
 from starfish.core.imagestack.imagestack import ImageStack
 from starfish.core.intensity_table.intensity_table import IntensityTable
-from starfish.core.types import Axes, Coordinates, Features
+from starfish.core.types import Axes, Coordinates, Features, SpotFindingResults
 
 
-def transfer_physical_coords_from_imagestack_to_intensity_table(
-        image_stack: ImageStack, intensity_table: IntensityTable
-) -> IntensityTable:
+def transfer_physical_coords_to_intensity_table(intensity_table: IntensityTable,
+                                                image_stack: Optional[ImageStack] = None,
+                                                spots: Optional[SpotFindingResults] = None,
+                                                ) -> IntensityTable:
     """
-    Transfers physical coordinates from an Imagestack's coordinates xarray to an intensity table
+    Transfers physical coordinates from either an Imagestack or SpotFindingResults to an
```

Comment:
Long term, will there be any workflows that extract physical coordinates from an ImageStack?  Or is that a stopgap measure until SpotFindingResults is produced everywhere?

Reply:
Yea I think we'll be able to remove the imagestack option once the full refactors done